src/blog-utils/StaticAddonCard/index.js

```diff
@@ -8,20 +8,30 @@ import AddonTitle from 'amo/components/AddonTitle';
 import GetFirefoxButton, {
   GET_FIREFOX_BUTTON_TYPE_ADDON,
 } from 'amo/components/GetFirefoxButton';
+import Rating from 'amo/components/Rating';
+import translate from 'amo/i18n/translate';
 import type { AddonType } from 'amo/types/addons';
+import type { I18nType } from 'amo/types/i18n';
 
 import './styles.scss';
 
-type Props = {|
+type PublicProps = {|
```

Comment:
Everywhere else we use `Props` to mean "public props" and `InternalProps` to mean "all of the props", but here you've reversed that and introduced `PublicProps`. Can we make this consistent with the rest of the code in the repo?

Reply:
sure